pkg/ccl/importccl/import_stmt.go

```diff
@@ -901,11 +900,14 @@ func doDistributedCSVTransform(
 	for i := 0; i < n; i++ {
 		row := rows.At(i)
 		name := row[0].(*tree.DString)
-		size := row[1].(*tree.DInt)
```

Comment:
Is this field used anymore? If not should it be removed?

Reply:
Ah, yeah, I forgot we didn't care about wire compat. Done.